lib/fluminus_cli.ex

```diff
@@ -241,6 +241,18 @@ defmodule FluminusCLI do
     end
   end
 
+  @spec rename_wrapper(Path.t(), Path.t()) :: :ok | {:error, Elixir.File.posix()}
+  defp rename_wrapper(source, destination) do
+    case Elixir.File.rename(source, destination) do
+      {:error, :exdev} ->
+        Elixir.File.cp(source, destination)
```

Comment:
Prolly you wanna do something like this instead:
```elixir
with :ok <- Elixir.File.cp(source, destination),
     :ok <- Elixir.File.rm(source, destination) do
  :ok
else
  error -> error
end
```

So that you'll fail early in the case of cp failure, and return the error instead of just swallowing the error and continuing with the `Elixir.File.rm/2`

Reply:
Or just one more nested case is ok also